### ISSUE TITLE
Unable to import Resource Manager from virtual-kubelet 1.0

### ISSUE THREAD

**Issue by gabrielhartmann** [virtual-kubelet/node-cli#1]
The package `github.com/virtual-kubelet/virtual-kubelet/manager` has been moved to `github.com/virtual-kubelet/internal/manager`.  This project currently has dependencies on that now internal package.

```bash
$ git diff go.mod
diff --git a/go.mod b/go.mod
index fb5ee7e..2865e9f 100644
--- a/go.mod
+++ b/go.mod
@@ -10,7 +10,7 @@ require (
        github.com/sirupsen/logrus v1.4.2
        github.com/spf13/cobra v0.0.5
        github.com/spf13/pflag v1.0.3
-       github.com/virtual-kubelet/virtual-kubelet v0.11.1
+       github.com/virtual-kubelet/virtual-kubelet v1.0.0
        go.opencensus.io v0.20.2
        k8s.io/api v0.0.0-20190222213804-5cb15d344471
        k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
```

```bash
$ go build
go: finding github.com/virtual-kubelet/virtual-kubelet/manager latest
provider/store.go:6:2: unknown import path "github.com/virtual-kubelet/virtual-kubelet/manager": cannot find module providing package github.com/virtual-kubelet/virtual-kubelet/manager
```

**Comment by cpuguy83** [comment]
Thanks, yes we should either do something other than resource manager or bring that in here.

**Comment by gabrielhartmann** [comment]
This also affects the Fargate provider: https://github.com/virtual-kubelet/aws-fargate/blob/master/cmd/virtual-kubelet/commands/root/root.go#L109

**Comment by cpuguy83** [comment]
This probably affects all providers since they were using resource manager.

**Comment by gabrielhartmann** [comment]
Addressed in #2
